### PR TITLE
[TS] LPP-47933 > LPS-175501 The image version has been changed, but the image remains the same

### PR DESCRIPTION
**Pull request by locpham97**
Hi team,
Please help me review this ticket.
https://issues.liferay.com/browse/LPS-175501
Thanks,
Loc
cc: @ces-huynguyen

### CODE REVIEW

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

File: modules/dxp/apps/document-library-opener/document-library-opener-onedrive-web/src/main/java/com/liferay/document/library/opener/onedrive/web/internal/service/DLOpenerOneDriveDLAppServiceWrapper.java [hunk 1]
```diff
@@ -171,8 +171,8 @@ public FileEntry updateFileEntryAndCheckIn(
 
 			return super.updateFileEntryAndCheckIn(
 				fileEntryId, sourceFileName, mimeType, title, urlTitle,
-				description, changeLog, dlVersionNumberIncrease, null,
-				expirationDate, revisionDate, serviceContext);
+				description, changeLog, dlVersionNumberIncrease, inputStream,
```
Comment (AliciaGarciaGarcia) [inline modules/dxp/apps/document-library-opener/document-library-opener-onedrive-web/src/main/java/com/liferay/document/library/opener/onedrive/web/internal/service/DLOpenerOneDriveDLAppServiceWrapper.java:174]: can you make the same on line 181
`return super.updateFileEntry(`

Thanks!

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/document-library-opener/document-library-opener-onedrive-web/src/main/java/com/liferay/document/library/opener/onedrive/web/internal/service/DLOpenerOneDriveDLAppServiceWrapper.java (hunk 1 above)
Comment (ces-huynguyen) [reply modules/dxp/apps/document-library-opener/document-library-opener-onedrive-web/src/main/java/com/liferay/document/library/opener/onedrive/web/internal/service/DLOpenerOneDriveDLAppServiceWrapper.java:174]: @AliciaGarciaGarcia updated. How about this implementation : 
if (_dlOpenerOneDriveManager.isConfigured(fileEntry.getCompanyId()) ||
			_dlOpenerOneDriveManager.isOneDriveFile(fileEntry)) {
			checkInFileEntry(
			fileEntryId, dlVersionNumberIncrease, changeLog, serviceContext);
		}		

		return super.updateFileEntry(
			fileEntryId, sourceFileName, mimeType, title, urlTitle, description,
			changeLog, dlVersionNumberIncrease, inputStream, size, expirationDate,
			revisionDate, serviceContext);

Will be shorter but a little bit of confuse.

Comment (AliciaGarciaGarcia) [reply modules/dxp/apps/document-library-opener/document-library-opener-onedrive-web/src/main/java/com/liferay/document/library/opener/onedrive/web/internal/service/DLOpenerOneDriveDLAppServiceWrapper.java:174]: I prefer the previous one. But up to you

Comment (ces-huynguyen) [reply modules/dxp/apps/document-library-opener/document-library-opener-onedrive-web/src/main/java/com/liferay/document/library/opener/onedrive/web/internal/service/DLOpenerOneDriveDLAppServiceWrapper.java:174]: We could keep it as it is. Thanks.

File: modules/dxp/apps/document-library-opener/document-library-opener-onedrive-web/src/main/java/com/liferay/document/library/opener/onedrive/web/internal/service/DLOpenerOneDriveDLAppServiceWrapper.java [hunk 2]
```diff
@@ -171,16 +171,16 @@ public FileEntry updateFileEntryAndCheckIn(
...
+				size, expirationDate, revisionDate, serviceContext);
 		}
 
 		checkInFileEntry(
 			fileEntryId, dlVersionNumberIncrease, changeLog, serviceContext);
 
 		return super.updateFileEntry(
 			fileEntryId, sourceFileName, mimeType, title, urlTitle, description,
-			changeLog, dlVersionNumberIncrease, null, 0, expirationDate,
+			changeLog, dlVersionNumberIncrease, inputStream, size, expirationDate,
```
Comment (AliciaGarciaGarcia) [inline modules/dxp/apps/document-library-opener/document-library-opener-onedrive-web/src/main/java/com/liferay/document/library/opener/onedrive/web/internal/service/DLOpenerOneDriveDLAppServiceWrapper.java:183]: SF Failing here :D can you Source Format it?

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

File: modules/dxp/apps/document-library-opener/document-library-opener-onedrive-web/src/main/java/com/liferay/document/library/opener/onedrive/web/internal/service/DLOpenerOneDriveDLAppServiceWrapper.java (hunk 2 above)
Comment (ces-huynguyen) [reply modules/dxp/apps/document-library-opener/document-library-opener-onedrive-web/src/main/java/com/liferay/document/library/opener/onedrive/web/internal/service/DLOpenerOneDriveDLAppServiceWrapper.java]: Yah, Source Formatted  :D

**Review by AliciaGarciaGarcia — APPROVED** [review]

File: modules/dxp/apps/document-library-opener/document-library-opener-onedrive-web/src/main/java/com/liferay/document/library/opener/onedrive/web/internal/service/DLOpenerOneDriveDLAppServiceWrapper.java (hunk 2 above)
Comment (AliciaGarciaGarcia) [reply modules/dxp/apps/document-library-opener/document-library-opener-onedrive-web/src/main/java/com/liferay/document/library/opener/onedrive/web/internal/service/DLOpenerOneDriveDLAppServiceWrapper.java]: thanks!! :)

**Comment by AliciaGarciaGarcia** [comment]
ci:test:onedrive

**Comment by AliciaGarciaGarcia** [comment]
ci:stop

**Comment by AliciaGarciaGarcia** [comment]
looks that ci is not responding, lets wait

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**